app/controllers/application_controller.rb

```diff
@@ -50,4 +50,9 @@ def get_commit
         @commit_url = "https://github.com/fsek/web/commit/%s" % @commit
       end
     end
+
+    def verify_admin
+      flash[:error] = t('the_role.access_denied')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.